<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1099</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listAccounts">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>256</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="expelStudent">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>400</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Исключить</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="descriptrion_1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>251</width>
      <height>351</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Описание (причины)</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listGroups">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>141</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="transferStudent">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>400</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Перевести в другую группу</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="score_text">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>460</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Количество баллов</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendScore_button_1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>500</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Отправить баллы</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Форма выпуска экзамена&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Release_Event_Button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Запустить</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="event_name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Название ивента</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="description_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>211</width>
      <height>121</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Описание</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>211</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listGroupsForTransfer">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>70</y>
      <width>141</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>20</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Группы перевода&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Описание (исключения и перевода в группу)</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="course_group">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>570</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="nameGroup">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>610</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="createGroup_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>690</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать группу</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Курс&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Название группы&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>650</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Факультет&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="school_account">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>610</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="score_account">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>650</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="password_account">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>570</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="username_account">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>650</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="name_student">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>570</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="fathername_student">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>650</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="surname_student">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>610</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="createAccount_button">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>690</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать аккаунт</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>286</x>
      <y>580</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Факультет (кто создает)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>620</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Школа</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>650</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Никнейм</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>580</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Пароль</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>580</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>610</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Фамилия</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>650</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Отчество</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>620</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Роль</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>660</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Очки</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>12</y>
      <width>241</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px solid black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>271</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>450</y>
      <width>271</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>251</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>560</y>
      <width>821</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="role_account">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>610</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Студент</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Учитель</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Администратор</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="faculty_by_creater_account">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>570</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="faculty_group">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>650</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <zorder>label_17</zorder>
   <zorder>label_16</zorder>
   <zorder>label_20</zorder>
   <zorder>label_19</zorder>
   <zorder>label_18</zorder>
   <zorder>listAccounts</zorder>
   <zorder>expelStudent</zorder>
   <zorder>descriptrion_1</zorder>
   <zorder>listGroups</zorder>
   <zorder>transferStudent</zorder>
   <zorder>score_text</zorder>
   <zorder>sendScore_button_1</zorder>
   <zorder>label</zorder>
   <zorder>Release_Event_Button</zorder>
   <zorder>event_name</zorder>
   <zorder>description_2</zorder>
   <zorder>listView</zorder>
   <zorder>listGroupsForTransfer</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>course_group</zorder>
   <zorder>nameGroup</zorder>
   <zorder>createGroup_button</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>school_account</zorder>
   <zorder>score_account</zorder>
   <zorder>password_account</zorder>
   <zorder>username_account</zorder>
   <zorder>name_student</zorder>
   <zorder>fathername_student</zorder>
   <zorder>surname_student</zorder>
   <zorder>createAccount_button</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>label_12</zorder>
   <zorder>label_13</zorder>
   <zorder>label_14</zorder>
   <zorder>label_15</zorder>
   <zorder>role_account</zorder>
   <zorder>faculty_by_creater_account</zorder>
   <zorder>faculty_group</zorder>
   <zorder>dateTimeEdit</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1099</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Форма создания группы</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Форма создания клиента</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
